doc/nrf/gs_installing.rst

```diff
@@ -235,19 +254,19 @@ For more information about the repository and development model, see the :ref:`d
 
 To clone the repositories, complete the following steps:
 
-1. Create a folder named ``ncs``.
+1. Create a folder named :file:`ncs`.
    This folder will hold all |NCS| repositories.
-#. Open a command window in the ``ncs`` folder.
-#. Determine what revision of the |NCS| you want to work with.
+#. Open a command window with administrator rights in the :file:`ncs` folder.
```

Comment:
Have you confirmed this? I really don't think you need administrator rights to download the NCS code.
And if you do this with administrator rights, you might not be able to access the files without admin rights afterwards - and it is very bad practice to work with admin rights by default.